bin/smack.py

```diff
@@ -141,6 +144,9 @@ def arguments():
   verifier_group.add_argument('--loop-limit', metavar='N', default='1', type=int,
     help='upper bound on minimum loop iterations [default: %(default)s]')
 
+  verifier_group.add_argument('--context-switches', metavar='k', default='1', type=int,
```

Comment:
`--context-bound` sounds like a fine solution - works for me.  Let's go with this.

I agree with you that `--context-switches` is mislabeled if its default value for sequential programs is 1.

I'll fix this in my next commit.